app/src/main/assets/shared_error_style.css

```
@@ -90,10 +105,14 @@ hr {
 }
 
 .buttonSecondary {
-  background-color: var(--secondary-button-color);
+  background-color: transparent;
   color: var(--secondary-button-text-color);
 }
 
+.buttonSecondary {
```

needs the `::after` :) 

LGTM once this is fixed!
```suggestion
.buttonSecondary::after {
```